src/providers/firebase-messaging.ts

```diff
@@ -0,0 +1,22 @@
+import * as firebase from 'firebase-admin';
+import {Message} from 'firebase-admin/messaging';
+import {ProviderInterface} from '../provider.interface';
+import {ProviderType} from '../provider-type';
+
+export class FirebaseMessaging implements ProviderInterface {
+	client: firebase.messaging.Messaging;
+	name = 'firebase-messaging';
+	type = ProviderType.MOBILE_PUSH;
+
+	constructor(certPath: string) {
```

Comment:
please also make certPath a parameter above like we have seen int `twilio-sendgrid`